Frameworks/UIKit/UIButton.mm

```diff
@@ -617,6 +618,25 @@ - (void)touchesMoved:(NSSet*)touchSet withEvent:(UIEvent*)event {
     WUXIPointerRoutedEventArgs* routedEvent = [event _touchEvent]->_routedEventArgs;
     [routedEvent setHandled:NO];
 
+    // We're assuming multitouchenabled = NO; it's a button after all.
+    CGPoint point = [[touchSet anyObject] locationInView:self];
+    BOOL currentTouchInside = [self pointInside:point withEvent:event];
+
+    UIControlState newState;
+
+    if (currentTouchInside) {
+        newState = _curState | UIControlStateHighlighted;
+    } else {
+        newState = _curState & ~UIControlStateHighlighted;
+    }
+
+    // Relayout when new state is different than old state
+    if(_curState != newState) {
+        _curState = newState;
+        [self setNeedsDisplay];
+        [self setNeedsLayout];
+    }
```

Comment:
we can call [self setHightlighted:currentTouchInside] to reuse some logic in UIControl.

---
In reply to: [96944367](https://github.com/Microsoft/WinObjC/pull/1731#discussion_r96944367) [](ancestors = 96944367,96943884)